WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeViewController.swift

```diff
@@ -269,19 +277,15 @@ extension AddAttributeViewController {
         guard let name = viewModel.newAttributeName else {
             return
         }
-        presentAddAttributeOptions(for: name)
+        presentAddAttributeOptions(attribute: .new(name: name))
     }
 
-    private func presentAddAttributeOptions(for newAttribute: String) {
-        let viewModel = AddAttributeOptionsViewModel(product: self.viewModel.product, attribute: .new(name: newAttribute))
-        let addAttributeOptionsVC = AddAttributeOptionsViewController(viewModel: viewModel)
-        navigationController?.pushViewController(addAttributeOptionsVC, animated: true)
-    }
-
-    private func presentAddAttributeOptions(for existingAttribute: ProductAttribute) {
-        let viewModel = AddAttributeOptionsViewModel(product: self.viewModel.product, attribute: .existing(attribute: existingAttribute))
-        let addAttributeOptionsVC = AddAttributeOptionsViewController(viewModel: viewModel)
-        navigationController?.pushViewController(addAttributeOptionsVC, animated: true)
+    /// Presents `AddAttributeOptionsViewController` and passes the same `onCompletion` closure, for our presenterVC  to handle.
+    ///
+    private func presentAddAttributeOptions(attribute: AddAttributeOptionsViewModel.Attribute) {
```

Comment:
What do you think of keeping `for` as the argument label for `attribute` here? As in:

```swift
private func presentAddAttributeOptions(for attribute: AddAttributeOptionsViewModel.Attribute) {
```

For me, that makes it easier to understand what's happening when you call it elsewhere, for example in line 187 it could read like a sentence:

```swift
presentAddAttributeOptions(for: .existing(attribute: attribute))
```

Reply:
Thanks for suggesting this! English has always been a struggle for me!